modules/social_features/social_group/social_group.module

```diff
@@ -246,6 +249,15 @@ function social_group_preprocess_group(array &$variables) {
 
   $account = \Drupal::currentUser();
 
+  // Prevent access to mute group notifications if a user is not a group member
+  // and show that link for specific regions only.
+  $allowed_regions = [
```

Comment:
Why is this necessary? 🤔 
Thats why we have our theme and view modes right? they dictate if a flag is rendered on a page in a view mode.

Imagine someone having a subtheme without these regions, or someone updating the config for a view mode and wanting to add it to a different region?

Reply:
You're right, I've moved it into view display settings.